src/packages/auth/src/auth-utils.ts

```diff
@@ -301,6 +301,12 @@ export async function checkAuthorization<G = unknown>(
 
 		if (isEntityMetadata(relatedEntityMetadata)) {
 			// Now we have a related entity, let's check and make sure we have permission
+			
+			// If a to-many relationship is being reset to an empty array, we don't need to check permissions
```

Comment:
This is a fix I found I needed when submitting various payloads. In this case you remove the last item from a to many relationship, and an empty array is passed to the server, which causes `requiredPermissionsForAction` to throw